runtime/v2/shim/shim_unix.go

```diff
@@ -38,7 +38,7 @@ import (
 // sub-reaper so that the container processes are reparented
 func setupSignals(config Config) (chan os.Signal, error) {
 	signals := make(chan os.Signal, 32)
-	smp := []os.Signal{unix.SIGTERM, unix.SIGINT, unix.SIGPIPE}
```

Comment:
This would also end up changing the behavior of the `delete` action. Is the intention to not capture these signals in that case and rely on the default behavior?

Reply:
I don't understand your question, we handle the signals elsewhere.

Reply:
The `chan os.Signal` which gets returned by this function is also used in the `delete` action. My understanding is that would change the the signal behavior for that action. That action does not use call the `serve` function. That action is intended to be short lived so not sure that it matters either way.

Reply:
Removed this change.